### PR TITLE
moving content for volume mounts from cf docs into 2.11

### PR DESCRIPTION
**Pull request by shalako**
- consistency of headings
- moved overview to top
- added missing spec for volume mounts field in bind response

### CODE REVIEW

**Comment by duglin** [comment]
LGTM

**Comment by angarg12** [comment]
LGTM

**Comment by zrob** [comment]
LGTM

**Comment by shalako** [comment]
One more needed

**Comment by mattmcneeney** [comment]
@shalako I think this might need a rebase? It seems to be changing a lot of unrelated lines

**Comment by avade** [comment]
Yeah @shalako the diff is now showing lots of changes, not just volume mount things.

**Comment by shalako** [comment]
@avade All the commits appear to be intentional. Since I'm no other changes have been made to the 2.11 branch since April 6th, I don't expect there's anything to rebase.

It's true there a few additional fixes to make version 2.11 of the docs match our recent formatting conventions.

File: spec.md [hunk 1]
```diff
@@ -614,47 +601,84 @@ $ curl http://username:password@broker-url/v2/service_instances/:instance_id/ser
...
-        "port": 3306,
-        "database": "dbname"
+##### Volume Mounts Object
+
+| Response Field | Type | Description |
+| --- | --- | --- |
+| driver* | string | Name of the volume driver plugin which manages the device. |
+| container_dir* | string | The path in the application container onto which the volume will be mounted. This specification does not mandate what action the platform is to take if the path specified already exists in the container. |
+| mode* | string | "r" to mount the volume read-only or "rw" to mount it read-write. |
+| device_type* | string | A string specifying the type of device to mount. Currently the only supported value is "shared".  |
```
Comment (mattmcneeney) [inline spec.md]: This might fail the travis checks due to the double space before the `|`.

**Comment by shalako** [comment]
Fixed spaces; Travis passed. Needs one more.

**Comment by mattmcneeney** [comment]
LGTM

**Comment by mattmcneeney** [comment]
Re LGTMs please from @duglin @angarg12 @zrob @avade

**Comment by angarg12** [comment]
LGTM

**Comment by duglin** [comment]
LGTM

**Comment by duglin** [comment]
@zrob @avade @vaikas-google @fmui one more review needed

**Comment by fmui** [comment]
LGTM

**Pull request merged**